dialogue/qa_pipeline.py

```diff
@@ -159,6 +159,11 @@ def get_query_model(command_class):
         engine = models.db_connect()
         session = sessionmaker(bind=engine)()
         return models, session
+    if command_class == "Diagnosis": #FIXME
```

Comment:
???

Reply:
As long as the command_class is intended to represent module skills and not the modules themselves, I guess this is alright. Removing FIXME, as that was my concern.